### DISCUSSION TITLE
Ducumantation

### DISCUSSION THREAD

**Discussion by tomekl1978** [StealthChesnut/HA-FoxESS-Modbus#13] [Ideas]
Just recieved documentation from Foxess to RS485 port in t4 (g3) series inverter. 
[Monitoring Protocol VB1.02.00_20220127_ for TG3.pdf](https://github.com/StealthChesnut/HA-FoxESS-Modbus/files/8650177/Monitoring.Protocol.VB1.02.00_20220127_.for.TG3.pdf)

**Comment by StealthChesnut** [comment]
Thanks for sharing this. Did you get this through support? If so, was it China support or local in-country support?

**Reply by tomekl1978** [reply]
In Poland. I'm trying to get it working but i have a problem with connection to USR converter.

**Comment by zxldz** [comment]
I have the same PDF from Foxess. I have correctly connected T8 to RS485 Waveshare. I cannot determine the addresses for which there is data sent by the inverter. I mean real-time information (page 5 and 6 PDF)

**Reply by mdk89** [reply]
> I have the same PDF from Foxess. I have correctly connected T8 to RS485 Waveshare. I cannot determine the addresses for which there is data sent by the inverter. I mean real-time information (page 5 and 6 PDF)

hi i have a foxess t8 and i want to hook it up after rs 485 via usr-w610.  What pins did you connect 1 and 2 or 3 and 4 to?

**Comment by gaszu** [comment]
I have from Foxess-Poland 2 documents  from Txx G first series and foxess cloud data structure - maybe it will be helpfull for U.
Foxess Txx G first series operates as master with own address 1 (no 247 slave) and generates data every 30 second - to the cloud and RS485 comm interface on pins 1 and 2.

Greets!
 
[Modbus Contral Communication Protocol_V101(1).pdf](https://github.com/StealthChesnut/HA-FoxESS-Modbus/files/8861269/Modbus.Contral.Communication.Protocol_V101.1.pdf)
[FoxESS.Modbus.On.Grid.solar.inverter.Modbus.Protocol.pdf](https://github.com/StealthChesnut/HA-FoxESS-Modbus/files/8861272/FoxESS.Modbus.On.Grid.solar.inverter.Modbus.Protocol.pdf)
